src/components/event/details/DetailsSuspectedCommits.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <DetailsBase
+    class="details-suspected-commit"
+    :expandShowed="commits.length > numberOfVisibleCommits"
+    @expandClicked="isMoreCommitsShown = !isMoreCommitsShown"
+  >
+    <template #header> {{ $t("event.suspectedCommits") }} </template>
+    <template #content>
+      <div
+        v-for="(commit, index) in filteredCommits"
+        :key="index"
+        class="event-details__content-block details-suspected-commit__content-block"
+        data-ripple
+      >
+        <div class="details-suspected-commit__left">
+          <div class="details-suspected-commit__left-title">
+            {{ commit.title }}
+          </div>
+          <div>
```

Comment:
do we need this extra div?

Reply:
I have implemented this
![120048369-94543d80-c01f-11eb-8c0c-7d9c88874457](https://user-images.githubusercontent.com/23169768/121537247-61e41080-ca21-11eb-8197-c38a3a498428.png)
and used flex for the same.
but I can also implement without flex in that case we don't need `div` but still need some `heading tag` to contain it.
So let me know.

Reply:
you can check [here](https://github.com/codex-team/hawk.garage/blob/d631e9c5e98a9e161d372c1cf9e3dc6586493f72/src/components/event/details/DetailsSuspectedCommits.vue#L124)
I am using the 
  
      .details-suspected-commit {
          &__content-block {}
          &__left > div {}
          &__left {
            &-title {}
            &-image {}
            &-author {}
            &-relative-time {}
          }
          &__right {
            &-hash-block {}
            &-hash {}
          }
        }